src/scripts/safari/Safari.ts

```diff
@@ -347,7 +347,9 @@ class Safari {
     }
 
     public static spawnItemCheck() {
-        if (Rand.boolean()) {
+        const baseChance = 0.4;
+        const levelModifier = (Safari.safariLevel() - 1) / 100;
+        if (Rand.chance(baseChance + levelModifier)) {
```

Comment:
Starting with Level 62 this would reduce your chance for an item compared to the previous Level

Reply:
Maximum level is 40, though. I do not think we plan on adding more levels as that one is already long to reach.